lib/import.php

```diff
@@ -103,10 +117,20 @@ protected function commit( $commit ) {
 				continue;
 			}
 
-			$posts[] = $post = $this->blob_to_post( $blob );
+			$post = $this->blob_to_post( $blob );
```

Comment:
Because we're still iterating over the tree (see `$commit->tree()->blobs()` above), is the savings here primarily from making fewer DB calls?